renderers/src/main/java/com/pedrogomez/renderers/Renderer.java

```diff
@@ -50,9 +51,12 @@
    * be null in this method.
    * @param layoutInflater used to inflate the view.
    * @param parent used to inflate the view.
+   * @param selector used to manage selection when using with RecyclerView.
    */
-  public void onCreate(T content, LayoutInflater layoutInflater, ViewGroup parent) {
+  public void onCreate(T content, LayoutInflater layoutInflater, ViewGroup parent,
+      Selector<T> selector) {
```

Comment:
This is the only one _breaking change_. If someone was overriding the previous `onCreate` or calling it directly (I don't find a strong reason to do it) then they will have to update their code just a little bit.